CHANGELOG.md

```diff
@@ -1,3 +1,10 @@
+## v0.3.0 [2022-11-09]
+
+_What's new?_
+
+- New tables added
+  - [datadog_slo](https://hub.steampipe.io/plugins/turbot/datadog/tables/datadog_slo)
```

Comment:
Thanks @whume for adding the release note, but it is not necessary to add it as a part of table development. This change can be safely reverted 👍.